src/redux/api.ts

```diff
@@ -7,3 +12,18 @@ export const baseQuery = fetchBaseQuery({
     Accept: 'application/json',
   },
 })
+
+export const rtkQueryErrorLoggerMiddleware: Middleware =
+  (api: MiddlewareAPI) => (next) => (action) => {
+    if (isRejectedWithValue(action)) {
+      const errorMessage =
+        'data' in action.error
+          ? (action.error.data as { message: string }).message
+          : action.error.message
+
+      Logger.error('RTK-Query', errorMessage || 'An error occured', action)
+      api.dispatch(showError(ErrorMessages.FETCH_FAILED) as Action)
+    }
+
+    return next(action)
+  }
```

Comment:
Ah, we don't really want this generic error message anymore at this level.
Could we move it to be at the component level instead?

We now usually prefer showing inline errors, instead of the red banner at the top.
To avoid showing them from background tasks we have in the app, which could show up at inappropriate times (e.g. the user navigated to another screen, so the red banner is not directly relevant for something that's in the TX feed).

`FETCH_FAILED` is really just used for the TX feed. 
I know moving it to the component won't change the fact that it could still trigger an inappropriate times.
But at least we can update the behavior later from the component.

Reply:
@jeanregisser Gotcha, I will adjust this to the new desirable approach. 
It seems like `queryHelper` only uses red banner for error. Could you please point me at the example with the inline error?

Reply:
Ah, got it. I'll move it to the component.